graylog2-web-interface/src/pages/StreamPermissionErrorPage.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow strict
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+
+import { FetchError } from 'logic/rest/FetchProvider';
+
+import UnauthorizedErrorPage from './UnauthorizedErrorPage';
+
+type Props = {
+  error: FetchError,
+};
+
+const StreamPermissionErrorPage = ({ error }: Props) => {
+  const description = (
+    <>
+      <p>This resource includes streams you do not have permissions to.</p>
```

Comment:
Not 100% sure, but I think it should be `you do not have permissions _for_`

Reply:
I thought about this as well and found examples for both cases, but having a look at the oxford dictionary it is "permissions for somebody/something" / "permissions to do something".

Reply:
But then, it is `permissions for (streams)`, right?

Reply:
It is and I've changed it accordingly.